src/services/posts.rs

```diff
@@ -20,36 +22,93 @@ pub async fn manual_hello() -> impl Responder {
     HttpResponse::Ok().body("Hey there!")
 }
 
-#[post("/add_favorite")]
-pub async fn add_favorite(
+#[get("/restaurant")]
+pub async fn restaurant(
+    pool: web::Data<Pool<ConnectionManager<SqliteConnection>>>,
+) -> impl Responder {
+    let conn = pool.get().unwrap();
+
+    HttpResponse::Ok().json(Restaurant::get_all_resturants(&conn))
+}
+
+#[get("/restaurant/id{id}")]
```

Comment:
Shouldn't this be `/restaurant/{id}`?
Currently, you have to have to have the URL `/restaurant/id155` if you want to get ID 155.